doc/user-guide/src/environment-variables.md

```diff
@@ -14,13 +14,13 @@
   instead use a local mirror, or to test the binaries from the staging
   directory.
 
-- `RUSTUP_DIST_ROOT` (default: `https://static.rust-lang.org/dist`)
-  Deprecated. Use `RUSTUP_DIST_SERVER` instead.
+- ~~`RUSTUP_DIST_ROOT`~~ *deprecated* (default: `https://static.rust-lang.org/dist`)
+  Use `RUSTUP_DIST_SERVER` instead.
 
 - `RUSTUP_UPDATE_ROOT` (default `https://static.rust-lang.org/rustup`) Sets
   the root URL for downloading self-update.
 
-- `RUSTUP_IO_THREADS` *unstable* (defaults to reported cpu count). Sets the
+- `RUSTUP_IO_THREADS` *unstable* (defaults to reported cpu count) Sets the
```

Comment:
Dropping the `.` leads to poor prose.

Reply:
That's the style followed by the rest of the file. If you wish to change it, I'm happy to change all of them.

Reply:
Ugh, I hate it when we're already inconsistent!

I think the . present helps scanning the docs - please do use that style.

Reply:
Done.